src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -274,6 +274,15 @@ double SolverInterfaceImpl::initialize()
   Event                    e("initialize", precice::syncMode);
   utils::ScopedEventPrefix sep("initialize/");
 
+  PRECICE_DEBUG("Compressing provided meshes");
+  Event e1("mesh compression", precice::syncMode);
```

Comment:
I think we have not documented this properly, but there is some kind of consistency in event naming.
"mesh.compression" + name of the mesh etc.

Reply:
This was a placeholder, good point.